requirements.txt

```diff
@@ -1,4 +1,4 @@
-openwisp-users~=0.4.0
+openwisp-users~=0.4.1
```

Comment:
Same as above

Reply:
Will update to v0.5 directly once it is released